openedx/core/djangoapps/catalog/utils.py

```diff
@@ -491,13 +499,14 @@ def get_owners_for_course(course_uuid):  # lint-amnesty, pylint: disable=missing
     return []
 
 
-def get_course_uuid_for_course(course_run_key):
+def get_course_uuid_for_course(course_run_key, traverse_pagination=True):
```

Comment:
same as before - you shouldnt need `traverse_pagination` as an kwarg - it should always be passed into `get_api_data` as `False` here.